### PR TITLE
bcm_2711_bootloader_config.md - rpi-update files now available

### PR DESCRIPTION
**Pull request by andrum99**
Also, link directly to beta discussion thread, and some other minor changes.

### CODE REVIEW

**Comment by andrum99** [comment]
Ready for review.

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 1]
```diff
@@ -285,46 +285,48 @@ sudo reboot
...
+This is only available in the BETA release and requires updated firmware via [rpi-update](../../raspbian/applications/rpi-update.md). If you aren't already familiar with how to use a USB drive for the root filesystem then you probably want to wait until this feature is in the default Raspbian image.
 
-There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350)
+There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350).
```
Comment (lurch) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:290]: Perhaps "this forum thread" would be a better link-title than "STICKY: HOWTO: Move the filesystem to a USB stick/Drive" ?

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 2]
```diff
@@ -285,46 +285,48 @@ sudo reboot
...
-* Alternatively use `sudo rpi-update` to update the firmware on a Raspbian SD card install and copy the files from there.
+* Alternatively use `sudo rpi-update` to update the firmware on a Raspbian SD card install, then copy the files from there.
 * Copy these updates to the boot partition on the USB device. From now on `sudo rpi-update` can be used from within Raspbian on the USB boot device.
 * A Linux kernel update is not required. Raspbian has been tested using the 4.19 and 5.4 (32 and 64 bit) kernel.
 
 ### USB device compatiblity
 There's no explicit set of supported devices. Initially we recommend using a USB pen drive or SSD. Hard drives will probably require a powered HUB and in all cases you should verify that the devices work correctly from within Raspbian using an SD card boot.
 
-Please post interoperability reports (positive or negative) on the Raspberry Pi forums. 
+Please post interoperability reports (positive or negative) on [this thread](https://www.raspberrypi.org/forums/viewtopic.php?f=63&t=274595) on the Raspberry Pi General Discussion forum.
```
Comment (lurch) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:326]: Not sure if the addition of the "General Discussion" is really needed?

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 3]
```diff
@@ -285,46 +285,48 @@ sudo reboot
...
 sudo apt full-upgrade
+```
 
-# As root edit /etc/default/rpi-eeprom-update and select BETA releases
+* As root, edit `/etc/default/rpi-eeprom-update` and select BETA releases.
 
-# Install the BETA version of the bootloader and replace the current configuration settings to enable USB boot.
-# See BOOT_ORDER property if you wish to migrate the configuration by hand.
+* Install the BETA version of the bootloader and replace the current configuration settings to enable USB boot.
+See `BOOT_ORDER` property if you wish to migrate the configuration by hand.
```
Comment (lurch) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:305]: Does "if you wish to migrate the configuration by hand" need more detailed explanation? Or do we think that everyone that's experimenting with the USB boot BETA will understand what that means?

Comment (andrum99) [reply hardware/raspberrypi/bcm2711_bootloader_config.md:305]: I was trying not to change too much of the original text. I think giving folk a link to the precise forum thread should be enough in this instance. I would rather not add additional complication to the page.

**Comment by andrum99** [comment]
Addressed @lurch's review comments.

**Comment by lurch** [comment]
LGTM :+1: 
Any comments from @timg236 ?

**Comment by timg236** [comment]
LGTM

**Pull request merged**